csharp/tests/Valkey.Glide.IntegrationTests/TestConfiguration.cs

```diff
@@ -74,22 +77,38 @@ public TestConfiguration()
 
         _scriptDir = Path.Combine(projectDir, "..", "utils");
 
-        // Stop all if weren't stopped on previous test run
-        StopServer(false);
+        TLS = Environment.GetEnvironmentVariable("tls") == "true";
 
-        // Delete dirs if stop failed due to https://github.com/valkey-io/valkey-glide/issues/849
-        // Not using `Directory.Exists` before deleting, because another process may delete the dir while IT is running.
-        string clusterLogsDir = Path.Combine(_scriptDir, "clusters");
-        try
+        if (Environment.GetEnvironmentVariable("cluster-endpoints") is not null || Environment.GetEnvironmentVariable("standalone-endpoints") is not null)
         {
-            Directory.Delete(clusterLogsDir, true);
+            string? clusterEndpoints = Environment.GetEnvironmentVariable("cluster-endpoints");
+            CLUSTER_HOSTS = clusterEndpoints is null ? [] :
+                [.. clusterEndpoints.Split(',').Select(s => s.Split(':')).Select(s => (s[0], ushort.Parse(s[1])))];
+            string? standaloneEndpoints = Environment.GetEnvironmentVariable("standalone-endpoints");
+            STANDALONE_HOSTS = standaloneEndpoints is null ? [] :
+                [.. standaloneEndpoints.Split(',').Select(s => s.Split(':')).Select(s => (s[0], ushort.Parse(s[1])))];
+            _startedServer = false;
         }
-        catch (DirectoryNotFoundException) { }
+        else
+        {
+            // Stop all if weren't stopped on previous test run
+            StopServer(false);
+
+            // Delete dirs if stop failed due to https://github.com/valkey-io/valkey-glide/issues/849
+            // Not using `Directory.Exists` before deleting, because another process may delete the dir while IT is running.
+            string clusterLogsDir = Path.Combine(_scriptDir, "clusters");
+            try
+            {
+                Directory.Delete(clusterLogsDir, true);
+            }
+            catch (DirectoryNotFoundException) { }
 
-        // Start cluster
-        CLUSTER_HOSTS = StartServer(true);
-        // Start standalone
-        STANDALONE_HOSTS = StartServer(false);
+            // Start cluster
+            CLUSTER_HOSTS = StartServer(true, TLS);
+            // Start standalone
+            STANDALONE_HOSTS = StartServer(false, TLS);
```

Comment:
Why do we only start the Cluster?

Reply:
We start both. I'm missing your question, sorry.